project_template/src/components/Icon.js

```diff
@@ -1,6 +1,12 @@
 import { h } from "preact";
 
-export default ({ icon, ...props }) => {
+const Icon = ({ icon, ...props }) => {
   const iconAsset = require(`src/assets/icons/${icon}.svg`).default;
-  return <svg {...props}><use xlinkHref={iconAsset.url} /></svg>;
+  return (
+    <svg {...props}>
+      <use xlinkHref={iconAsset.url} />
+    </svg>
+  );
 };
+
+export default Icon;
```

Comment:
Does this really make a difference? With this change do you see the name `Icon` anywhere while debugging?

Reply:
Yes using the React dev tools, you will see `Icon` instead of `Unknown`

Reply:
Ah ok, I googled it and indeed this seems the recommended approach.
This apparently also influences the names that show up in stacktraces, so this is quite useful in that regard too.